tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+import logging
+from pathlib import Path
+
+# Disable most logging for unit tests.
```

Comment:
The unit test library doesn't handle this? Guess not.

Pytest does if a test fails it shows output otherwise it swallows it by default

When running on GHA we want all output on (so we don't need to re-run locally to diagnose every failure)